### ISSUE TITLE
Duplicate fullbleed container landmark regions

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#7578]
## Bug Description

On the canvas there are multiple containers in different sub trees with the same label: `[role=presentation][aria-label="Fullbleed area"]`

There might be others too beyond this one.

Generally speaking these should be unique on a page.

## Screenshots

![Screenshot 2021-05-19 at 00 32 32](https://user-images.githubusercontent.com/841956/118732016-df10d100-b839-11eb-9b77-a76287956c89.png)

## Additional Context

- Plugin Version: 1.7
- WordPress Version: 5.7
- Operating System: macOS
- Browser: Chrome

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
